core/templates/calendar.html

```diff
@@ -94,6 +94,7 @@
   </section>
 </body>
 <script>
+  var now = <%- date.getTime() %>;
   var allEvents = <%- JSON.stringify(allCalendarEvents) %>;
   var userName = "<%- user %>";

```

Comment:
If you prefix each of these with `window.`, it's clear that you're making global variables, and I think CodeClimate will be less fussy (for example, when you use `window.allEvents` in cal.js